torch/linalg/__init__.py

```diff
@@ -2036,3 +2036,5 @@
     >>> torch.dist(Q.transpose(-2, -1) @ Q, torch.eye(4))
     tensor(6.2158e-07)
 """)
+
+tensordot = torch.functional.tensordot
```

Comment:
`torch.linalg.tensordot` should be the primary function and `torch.functional.tensordot` use it.
Documentation for `torch.linalg.tensordot` should be in this file like for any other function from `linalg` module.